apps/api/src/models/dummy.ts

```diff
@@ -333,11 +336,11 @@ const DUMMY_ITEMS: ItemsType[] = [
   {
     id: 2,
     creatorId: 3,
-    title: 'The Lord of the Rings Trilogy Blu-ray Set',
-    description: 'Complete Blu-ray set of The Lord of the Rings trilogy, featuring extended editions of all three films with bonus content.',
-    imageUrl: 'https://example.com/lotr-blu-ray-set.jpg',
-    brand: 'Warner Bros.',
-    price: 9999, // in cents ($99.99)
+    title: 'Batman: The Dark Knight Returns Graphic Novel',
+    description: 'Classic graphic novel by Frank Miller featuring an older Bruce Wayne returning to the role of Batman.',
+    imageUrl: 'https://example.com/batman-dark-knight-returns.jpg',
+    brand: 'DC Comics',
+    price: 1999, // in cents ($19.99)
     isForSale: true,
     isForTrade: true,
   },
```

Comment:
This makes sense. The ItemCard I created is pretty simple: it has only a title, image, and description.

In the wireframes, the modal shows only a little more information, namely tags. So I'll display those.

Perhaps we can add an optional field, 'itemURL', that applies wishlists but not to collections.

Reply:
I actually haven't finished defining `DUMMY_ITEMS_TAGS` (underneath `DUMMY_ITEMS`). That would populate the join table that stores items' tags. These tags will be returned on an item query i.e. GET `/item/:itemId`, so expect a field like `tags: ["Star Wars", "Marvel"]` in the response.

`itemUrl` is probably necessary, i'll add that.

btw i'll be changing price to be 2-places decimal.